package/crds/sample.template.crossplane.io_mytypes.yaml

```diff
@@ -8,6 +8,10 @@ metadata:
 spec:
   group: sample.template.crossplane.io
   names:
+    categories:
+    - crossplane
+    - provider
+    - PROVIDER-NAME
```

Comment:
@negz @hasheddan It would be awkward if contributors copied this template and forgot to replace `PROVIDER-NAME`. So I wonder whether this PR is necessary.

Reply:
Hrm, I think there's a bunch of places where we assume the provider name is `template` and that folks will change it so this shouldn't be too bad. 🤔

Reply:
I found it useful to know which things I should change from the template. The string `template` is all over the place so it will be great to make it explicit that `template` means provider name.. which I am not 100% sure that it means that for every appearance of `template`. The same happens with `sample` in `sample.template.crossplane.io`